### DISCUSSION TITLE
Table's changes on APP Store/Play Store

### DISCUSSION THREAD

**Discussion by gfb-47** [simolus3/drift#1668] [Q&A]
Every time we change some columns on our tables, we have to run the build runner and re install the app right?

I wonder if when I release the app, my users will need to the same. I mean, does the app store/play store updates cover this changes on tables?

**Comment by davidmartos96** [comment]
The SQLite database holds the version of the schema (the tables).
With drift, in the migrator you need to override the onUpgrade callback to do the desired changes on the database so that the users don't need to reinstall the app

**Reply by simolus3** [reply]
Exactly this! If you want to change columns or databases between app versions, you'll have to write a bit of code to migrate between versions. Drift has some utilities to help you with that, you can see the [docs on migrations](https://drift.simonbinder.eu/docs/advanced-features/migrations/) for more information.
